### ISSUE TITLE
Document Rage and Dead Click 

### ISSUE THREAD

**Issue by bruno-garcia** [getsentry/sentry-docs#8324]
Main goals:

- [ ] The explanation for how Dead and Rage clicks are detected
- [ ] SDK options, such as `slowClickIgnoreSelectors` (https://github.com/getsentry/sentry-javascript/issues/9087)
- [ ] Built-in attempts to ignore false positives (e.g: https://github.com/getsentry/getsentry/pull/11728)
- [ ] Known false positives for dead click: Browser download
- [ ] If we can fit it in: Document good UX suggestions like loading a spinner on a paging component to avoid triggering Rage Click (if user doesn't see UI changes, it's bad UX)

Note we've added R+D Click here to product docs: https://docs.sentry.io/product/session-replay/replay-page-and-filters/

Basically: 
> Dead Clicks: User clicks on a and button tags that do not result in any page activity after 7 seconds (i.e. no HTML was added, removed, or updated; no visual changes were observed in the page).
> Rage Clicks: Five or more clicks on a dead element (it exhibits no page activity after 7 seconds.) Rage clicks are a subset of dead clicks.

And we link to that page from the [changelog announcement of the new cards](https://changelog.getsentry.com/announcements/top-selectors-with-rage-and-dead-clicks-in-session-replay).
